src/main/kotlin/org/rust/ide/annotator/RustHighlightingAnnotator.kt

```diff
@@ -31,6 +35,15 @@ class RustHighlightingAnnotator : Annotator {
 
         override fun visitTypeParam(o: RustTypeParamElement) {
             holder.highlight(o, RustColor.TYPE_PARAMETER)
+            o.typeParamBounds?.let {
+                holder.highlight(it.colon, RustColor.KEYWORD) // feels off
```

Comment:
I'd favor explicit lambda params here over `it`.  A bit hard to grok with `it` changing context so many times.